defaults/main.yml

```diff
@@ -163,7 +163,7 @@ db:
     pass: dvnsecret
     jdbcurl: 
     log_lock_waits: true
-    version: 9.6
+    version: 10
```

Comment:
Just noting that postgres is bumped from 9.x to 10.x

Reply:
Yes. The python-psycopg2 module shipped with RHEL/CentOS8 is not backwards-compatible with 9.6.